pello/pello_greeting.py

```diff
@@ -1,3 +1,6 @@
+from blessings import Terminal
```

Comment:
This will fail if the package is not installed. The point of having the extra is to have *optional* functionality. I.e. the tool must keep functioning without `blessings`.